### DISCUSSION TITLE
Why can't you delete discussions?

### DISCUSSION THREAD

**Discussion by Jakobud** [community/community#3608] [Discussions]
I believe that repo maintainers at this point are the only ones who can delete a discussion. Why can't individuals delete their own discussions? It would help to cut down on duplicate discussions.

Currently if you only allow the maintainers to delete the discussions then you are requiring too much of them. For a small project it's not a big deal but for a big project its yet another thing they have to maintain. And if it's ignored then the Discussions section is potentially filled with duplicates that will never get cleaned up.

If a user was able to delete his own discussions it would go a long way toward solving this problem.

**Comment by kielnino** [comment]
I was looking for the delete button too. My discussion-post on a project has lapsed and I would like to delete it.

**Comment by PranamBhat** [comment]
Check this out [https://github.com/github/feedback/discussions/2948#discussioncomment-205422](https://github.com/github/feedback/discussions/2948#discussioncomment-205422)
